spec/actors/hyrax/actors/file_set_actor_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+require 'rails_helper'
+# Load the override being tested
+require Rails.root.join('app/overrides/actors/hyrax/actors/file_set_actor_override.rb')
+
+RSpec.describe Hyrax::Actors::FileSetActor do
+  let(:file_set) { FactoryBot.create(:file_set) }
+  let(:user) { FactoryBot.create(:user) }
+  let(:file_path) { File.join(fixture_path, 'files', 'image.png') }
+  let(:file) { File.new(file_path) }
+  let(:relation) { :original_file }
+  let(:actor) { described_class.new(file_set, user) }
+  let(:file_actor)  { Hyrax::Actors::FileActor.new(file_set, relation, user) }
+
+  describe '#update_content' do
+    it 'calls ingest_file and returns ingest job' do
```

Comment:
```suggestion
    it 'calls perform_now and returns ingest job' do
```
I think the next test is the one that verifies ingest_file is called, so maybe it should be mentioned there? although that test is not actually expecting it to be called, just allowing.